source/common/umf_helpers.hpp

```diff
@@ -160,6 +161,39 @@ template <typename Type> umf_result_t &getPoolLastStatusRef() {
     return last_status;
 }
 
+/// @brief translates UMF return values to UR.
+/// This function assumes that the native error of
+/// the last failed memory provider is ur_result_t.
+ur_result_t umf2urResult(umf_result_t umfResult) {
+    if (umfResult == UMF_RESULT_SUCCESS) {
```

Comment:
nit: why not have this under the switch with the rest?

Reply:
I was inspired by the `ur2piResult()` on sycl repo and didn't bother changing it.
I've moved it to the switch statement now.